apps/mail/components/mail/mail-list.tsx

```
@@ -697,358 +459,390 @@ const Thread = memo(
+              </div>
+            ) : (
+              <div className="flex flex-1 flex-col" id="mail-list-scroll">
+                <VList
+                  count={filteredItems.length}
+                  overscan={5}
+                  className="style-scrollbar flex-1 overflow-x-hidden"
+                  children={vListRenderer}
+                />
+              </div>
+            )}
+          </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)